tsconfig.json

```diff
@@ -1,10 +1,63 @@
-// prettier-ignore
 {
```

Comment:
Basically tsconfig file contain information for compiler how to compile the files and what to include and exclude while compiling the code. So I have added all the options in this tsconfig file so that we can strictly add some conditions that we want developer to follow else error will come. Anytime we can change this config no issue.